sdk/defendereasm/Azure.ResourceManager.DefenderEasm/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please specify the release date by replacing `Unreleased` , format is `YYYY-MM-DD`